ansible/roles/microshift-start/tasks/main.yml

```diff
@@ -69,22 +126,29 @@
 - name: Add boot info to local file
   lineinfile:
     path: boot.txt
-    line: "Second boot: {{ runtime }}"
+    line: "First boot: {{ runtime }}"
     create: yes
   delegate_to: localhost
 
 - name: finish pbench capture
+  become: yes
```

Comment:
This code needs to be conditional so that we can run Prometheus stats for the start procedure.
It would be nice to have it moved to common tasks as well.

Reply:
I must be missing where this condition is. Can you point me to the line?

Reply:
This one is on line 146